src/ir/analysis/derive_partial_eq_or_partial_ord.rs

```diff
@@ -13,31 +14,39 @@ use ir::ty::TypeKind;
 use std::collections::HashMap;
 use std::collections::HashSet;
 
-/// An analysis that finds for each IR item whether partialeq or partialord cannot be derived.
+/// An analysis that finds for each IR item whether `PartialEq`/`PartialOrd`
+/// cannot be derived.
 ///
-/// We use the monotone constraint function `cannot_derive_partialeq_or_partialord`, defined as
-/// follows:
+/// We use the monotone constraint function
+/// `cannot_derive_partialeq_or_partialord`, defined as follows:
 ///
 /// * If T is Opaque and layout of the type is known, get this layout as opaque
 ///   type and check whether it can be derived using trivial checks.
-/// * If T is Array type, partialeq or partialord cannot be derived if the length of
-///   the array is larger than the limit or the type of data the array contains cannot derive
-///   partialeq or partialord.
+///
+/// * If T is Array type, `PartialEq` or partialord cannot be derived if the
+///   length of the array is larger than the limit or the type of data the array
+///   contains cannot derive `PartialEq` or partialord.
```

Comment:
Is `partialord` intentional  or just missed?

Reply:
Just missed; thanks!